src/CLI/README.MD

```diff
@@ -16,7 +16,7 @@ You invoke the scanner by running `AxeWindowsCLI.exe`. By default, this tool get
 If you run the tool without parameters, you'll be presented with the help screen. An example follows:
 
 ```
-AxeWindowsCLI 2.1.1
+AxeWindowsCLI 2.1.4
```

Comment:
Future note--we should bump this when we update package versions after each release. It would appear that we overlooked this in the last 3 releases ☹️